.github/robots.txt

```diff
@@ -0,0 +1,5 @@
+# This should disallow all search index crawling of the documentation.
+# we are using RTD *only* for PR previews, and not for hosting the actual docs.
+# So this should be added to the HTML output within readthedocs.
+User-agent: *
+Disallow: /
```

Comment:
It looks like we are generating this and RTD is serving this.

![image](https://github.com/executablebooks/mystmd/assets/913249/37735194-5f40-4af0-ac6c-0eeebf12e55b)

Also a bug.